src/models/EntityTerm.php

```diff
@@ -18,19 +17,28 @@ public function rules()
 		];
 	}
 
+	//return term model assigned to this entity
   public function getTerm()
   {
     return $this->hasOne(Term::className(), ['id' => 'term_id']);
   }
 
-	public function getNEntity()
+	//return short name of this entity model, example: Post, Host
+	public function getEntityShortName()
 	{
 		$name = explode("\\",$this->entity);
 		$result = end($name);
 		return $result;
 	}
 
-	public function getREntity()
+	//return base name of this entity model, example: \common\models\Post
+	public function getEntityBaseName()
+	{
+		return $this->entity;
+	}
+
```

Comment:
That one is useless we already have getEntity() and ->entity which is provide by the framework why do we need to create a new one ?